mobile/ios/lightning/LndReactModule.m

```diff
@@ -158,6 +160,25 @@ @implementation LndReactModule
     [fileMgr removeItemAtPath:confTarget error:nil];
     [fileMgr copyItemAtPath:lndConf toPath: confTarget error:nil];
 
+    NSString *logFile = [self.appDir stringByAppendingString:@"/logs/bitcoin/testnet/lnd.log"];
+    NSFileHandle *fileHandle = [NSFileHandle fileHandleForReadingAtPath:logFile];
+
+    dispatch_async(dispatch_get_main_queue(), ^{
+        [[NSNotificationCenter defaultCenter] addObserverForName:NSFileHandleDataAvailableNotification
+                                                          object:fileHandle
+                                                           queue:[NSOperationQueue mainQueue]
+                                                      usingBlock:^(NSNotification *n) {
+                                                          NSData *data = [fileHandle availableData];
+                                                          if ([data length] > 0) {
+                                                              NSString *s = [NSString stringWithUTF8String:[data bytes]];
+                                                              [self sendEventWithName:logEventName body:@{ logEventDataKey:s }];
```

Comment:
So I would listen to this event like so ?

```js
lndEvent.addListener('logs', string => {
  console.log(string);
});
```

Reply:
Possibly like this 
```JS
lndEvent.addListener('logs', data => {
  console.log(data.logs);
});
```

but that can easily be changed :)